tests/e2e/parameters/infrastructure.py

```diff
@@ -0,0 +1,103 @@
+from pyclbr import Function
+
+from aws_cdk import CfnOutput
+from aws_cdk import aws_appconfig as appconfig
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_ssm as ssm
+
+from tests.e2e.utils.infrastructure import BaseInfrastructure
+
+
+class ParametersStack(BaseInfrastructure):
+    def create_resources(self):
+        functions = self.create_lambda_functions()
+        self._create_parameter_string(function=functions["ParameterStringHandler"])
+        self._create_app_config(function=functions["ParameterAppconfigFreeformHandler"])
+
+    def _create_parameter_string(self, function: Function):
+        parameter = ssm.StringParameter(
+            self.stack, id="string_parameter", parameter_name="sample_string", string_value="Lambda Powertools"
+        )
+
+        parameter.grant_read(function)
+
+        CfnOutput(self.stack, "ParameterString", value=parameter.parameter_name)
+        CfnOutput(self.stack, "ParameterStringValue", value=parameter.string_value)
+
+    def _create_app_config(self, function: Function):
+
+        cfn_application = appconfig.CfnApplication(self.stack, id="appconfig-app", name="appe2e", description="appe2e")
+        CfnOutput(self.stack, "AppConfigApplication", value=cfn_application.name)
+
+        cfn_environment = appconfig.CfnEnvironment(
+            self.stack, "appconfig-env", application_id=cfn_application.ref, name="enve2e", description="enve2e"
+        )
+        CfnOutput(self.stack, "AppConfigEnvironment", value=cfn_environment.name)
+
+        cfn_deployment_strategy = appconfig.CfnDeploymentStrategy(
+            self.stack,
+            "appconfig-deployment-strategy",
+            deployment_duration_in_minutes=0,
+            final_bake_time_in_minutes=0,
+            growth_factor=100,
+            name="deploymente2e",
```

Comment:
I'm 98% certain this won't be a problem, but I wonder if AppConfig allows multiple Deployment Strategies with the same name in the event of parallel tests running (3.7-3.9).

If we do see any CI error, we can easily use our builder to generate random values.